expr/context.go

```diff
@@ -10,7 +10,7 @@ import (
 // of Zed values both for let-style temporary variables accessible via
 // the scope and for allocating results.
 type Context interface {
-	Scope() []zed.Value
+	Vars() []zed.Value
```

Comment:
Maybe update the doc comment to make clear that the `Vars` method provides access to the scope?

Reply:
Yes! Definitely a good idea to leave a meaningful comment here as this somewhat complicated interface evolves.